common/src/main/java/io/homeassistant/companion/android/common/sensors/AppSensorManagerBase.kt

```diff
@@ -137,21 +136,7 @@ class AppSensorManager : SensorManager {
         }
     }
 
-    private fun updateCurrentVersion(context: Context) {
-
-        if (!isEnabled(context, currentVersion.id))
-            return
-
-        val state = BuildConfig.VERSION_NAME

```

Comment:
This is the only part that is unique to the module/flavor, so to limit duplicate code I propose not removing the entire function here but getting this value from a function

```kotlin
// in updateCurrentVersion
val state = getCurrentVersion()

// defined as
abstract fun getCurrentVersion(): String
```

which is implemented in the app/wear `AppSensorManager`s:

```kotlin
override fun getCurrentVersion(): String = BuildConfig.VERSION_NAME
```

Reply:
that is a lot cleaner, thank you 🙏